src/coreimage.cs

```diff
@@ -941,6 +942,14 @@ interface CIFilterInputKey {
 		[Since (7,0)]
 		[Field ("kCIInputExtentKey", "+CoreImage")]
 		NSString Extent  { get; }
+
+		[iOS (11,0), TV (11,0), Mac (10,13)]
```

Comment:
@VincentDondain make are to run intro on mac/32 bits on 10.13 (in case they are only available on 64 bits).
The jenkins bots are on 10.12 and it's better verified before reaching wrench.